pygpudrive/env/README.md

```diff
@@ -12,7 +12,11 @@ Configure the environment using the basic settings in `config`:
 config = EnvConfig()
 ```
 
-This `config` all environment parameters.
+This `config` all environment parameters. In `config`, you can choose the `dynamics_model` which converts action features `e.g. (accel, steer), (dx, dy, dyaw)` and method for expert trajectory moves from [waymax](https://github.com/waymo-research/waymax/blob/main/waymax/dynamics/).
```

Comment:
Nit: Please add a bit more detail:

This `config` contains all environment parameters. A dynamics model computes the successor state (e.g. position, yaw, velocity) of one or more objects given an action (e.g. steering and acceleration). 

The following dynamics models are currently implemented:
- Classic: [TODO: please define]
- InvertibleBicycleModel. A kinematically realistic model using a 2D action (acceleration, steering curvature).  (taken from [source](https://github.com/waymo-research/waymax/tree/main/waymax/dynamics))

- DeltaLocal. A position-based model using a 3D action (dx, dy, dyaw) representing the displacement of an object relative to the current position and orientation. This model does not check for infeasible actions, and large displacements may lead to unrealistic behavior. (taken from [source](https://github.com/waymo-research/waymax/tree/main/waymax/dynamics))